README.md

```diff
@@ -966,8 +966,8 @@ Most of utilities are found by native API. Others advanced functions could be ch
 
   // Native
   // But concat function doesn't remove duplicate items.
-  function merge(...args) {
```

Comment:
I would still like to retain ES2015 rest params over arguments, since arrow functions etc... are used throughout.